examples/inverter_example.py

```diff
@@ -26,6 +24,21 @@ def main():
 
     predictions_with_recent_pv_df["power_kw_no_live_pv"] = predictions_df["power_kw"]
 
+    # Get the directory of the current script
```

Comment:
Nice that you have added the functionality to save to csv in, I would perhaps add a parameter and if statement in main to make this saving to csv optional? 

Then you could perhaps use the `typer` library https://typer.tiangolo.com/ to be able to set this parameter via the command line option e.g. if you wanted to save the outputs to csv you could run

`python examples/inverter_example.py --save-outputs=True`

Reply:
Yeah this sounds good, will do this next

Reply:
Hey @Sukh-P , I have made these changes and tested them, and i've also deleted that csv file from the repo for now, it's still stored locally, will merge it as one big csv file later when we have enough data